modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/GridDiscoveryManager.java

```diff
@@ -1598,10 +1598,10 @@ private void topologySnapshotMessage(
 
         clo.apply(summary.toString());
 
-        ClusterNode currCrd = discoCache.coordinator();
+        ClusterNode currCrd = discoCache.oldestServerNode();
```

Comment:
Should we use `oldestAliveServerNode` instead of `oldestServerNode` here? Not sure, just asking.

Reply:
Fixed.